robocup_ws/src/Planner/src/brachistochrone.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import numpy as np
+from sympy import Symbol
+from scipy.optimize import newton
+from scipy.optimize import fsolve
+import matplotlib.pyplot as plt
+from sympy import lambdify
+from scipy.integrate import quad
+import sympy
+
+def func(x, f, fp):
```

Comment:
Nice :) 
you can think about wrapping it into a static class
```
class Brachistochrone:
   @staticmethod
   def func(x, f, fp):
        ....
 
   def parabola(...):
      ....

```
Sth similar you can find here:
https://github.com/szgula/UofG_Robotics_TDP/blob/a7fddf38f5438ca2089b1e7bdb62ef6a77113c7a/robocup_ws/src/Planner/src/BasicCommonActions/plan_supporting_functions.py#L6